app/javascript/App/bundles/NewsfeedPage/NewsfeedPage.tsx

```diff
@@ -222,14 +224,15 @@ class NewsfeedPage extends React.Component<IProps, IState> {
             <>{/* <NewsListHeader />
               <NewsList /> */}</>
           }
+          // @ts-ignore
           rightSection={<BodySection />}
           drawerSection={<CoinListDrawer />}
         />
       )
     } else {
       return (
         <LayoutDesktop
-          leftSection={<CoinListWrapper />}
+          leftSection={<CoinListWrapper loggedIn={this.props.loggedIn} />}
```

Comment:
As per discussion in slack with @hanchang the `loggedIn` will be reverted to `user` prop in some components